### ISSUE TITLE
AOSP style not working with configuration in pom.xml

### ISSUE THREAD

**Issue by sachin086** [Cosium/maven-git-code-format#21]
I was trying to change the code style from GOOGLE style to AOSP.

I've tried the below given config in pom.xml but its not getting formatted to 4-space indentation.

```
<plugin>
   <groupId>com.cosium.code</groupId>
   <artifactId>maven-git-code-format</artifactId>
   <version>1.20</version>
   <executions>
      <execution>
         <goals>
            <goal>install-hooks</goal>
         </goals>
      </execution>
   </executions>
   <configuration>
      <googleJavaFormatOptions>
         <aosp>true</aosp>
      </googleJavaFormatOptions>
   </configuration>
</plugin>
```

With this config its not getting formatted to 4-space style. 
I've used **mvn git-code-format:format-code -DglobPattern=**/*** to format and tried a commit as well.
Am I missing anything? Please help.

**Comment by reda-alaoui** [comment]
Hi @sachin086 ,

As you can see in https://github.com/Cosium/maven-git-code-format/commit/8cf5b21b1f90125741187911633caa23d6d55911, google java format options customization are only available since version *1.28*.

**Comment by sachin086** [comment]
All right. Thanks 👍
